quantum/quantum_keycodes.h

```diff
@@ -889,3 +893,8 @@ enum quantum_keycodes {
 #define DM_RSTP DYN_REC_STOP
 #define DM_PLY1 DYN_MACRO_PLAY1
 #define DM_PLY2 DYN_MACRO_PLAY2
+
+// One Shot toggle
+#define OST_TOG ONESHOT_TOGGLE
+#define OST_ON  ONESHOT_ENABLE
+#define OST_OFF ONESHOT_DISABLE
```

Comment:
```suggestion
#define OS_TOGG ONESHOT_TOGGLE
#define OS_ON   ONESHOT_ENABLE
#define OS_OFF  ONESHOT_DISABLE
```
Where possible the prefix should be two characters (IMO).

Reply:
Sounds good to me.